src/handler.js

```diff
@@ -88,21 +88,14 @@ module.exports.receptionist = (event, context, callback) => {
         response.body = jsonBody.challenge;
     // respond to bot mentions and interactive message callbacks from Slack. Ignore http_timeout retries
     } else if (( jsonBody.type === "interactive_message" || jsonBody.event.text.includes(`@${client.botID}`) ) && !timeoutRetry){
-        // asynchronously call event Lambda
-        lambda.invoke({
-            FunctionName: 'poet-bot-dev-event',
-            InvocationType: 'Event',
-            Payload: JSON.stringify(event, null, 2)
-            }, function(error, data) {
-            if (error) {
-                console.log("Invoke error: " + error);
-//                context.done('error', error);
-            }
-            if(data.Payload){
-                console.log("Invoke success: " + data.Payload);
-                context.succeed(data.Payload)
-            }
-        });
+        // publish the event to the SNS topic for the event lambda to pickup
+        console.log('Publishing event to SNS queue. ARN: ', TOPIC_ARN);
+        sns.publish({
+            Message: JSON.stringify(event),
+            TopicArn: TOPIC_ARN
+        }).promise()
```

Comment:
Out of interest, why not use `await` here?

Reply:
Good point, I've converted it to use await.